uwblueprint/childrens-aid-society
# Create modal new provider

## Notion ticket link

[Create New Provider Modal](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* used implementation from https://github.com/uwblueprint/childrens-aid-society/pull/104
* added button on homepage to new provider



## Steps to test
1. go into this branch
2. go to homepage and scroll all the way to the bottom
3. click the new providers modal button to see the modal in action
![Screen Shot 2022-11-13 at 12 03 15 PM](https://user-images.githubusercontent.com/77185514/201534267-960a1515-1112-4551-bc70-f3452f3a06ce.png)


## What should reviewers focus on?
* if the designs match
* where the modal should exist instead of the homepage
![Screen Shot 2022-11-13 at 12 03 15 PM](https://user-images.githubusercontent.com/77185514/201534267-960a1515-1112-4551-bc70-f3452f3a06ce.png)


## Checklist
